examples/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+alias(
```

Comment:
I'm a little uncomfortable with the libsnark examples being exposed so high up in the snarky repo. Can we move this down or remove these aliases?